test/EventStore.Client.PersistentSubscriptions.Tests/SubscriptionToAll/get_info.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace EventStore.Client.SubscriptionToAll {
+	public class get_info : IClassFixture<get_info.Fixture> {
+		private readonly Fixture _fixture;
+		private const string GroupName = nameof(get_info);
+		private static readonly PersistentSubscriptionSettings _settings = new(
+			resolveLinkTos: true,
+			startFrom: new Position(2, 1),
+			extraStatistics: true,
+			messageTimeout: TimeSpan.FromSeconds(9),
+			maxRetryCount: 11,
+			liveBufferSize: 303,
+			readBatchSize: 30,
+			historyBufferSize: 909,
+			checkPointAfter: TimeSpan.FromSeconds(7),
+			minCheckPointCount: 13,
+			maxCheckPointCount: 999,
+			maxSubscriberCount: 500,
+			namedConsumerStrategy: SystemConsumerStrategies.Pinned
+		);
+		
+		public get_info(Fixture fixture) {
+			_fixture = fixture;
+		}
+
+		[Fact]
+		public async Task returns_expected_result() {
```

Comment:
**SUGGESTION:** I think that it'd be good to check negative scenarios, like getting a status for a not-existing persistent subscription. A similar comment applies to other tests.